programs/svm-spoke/src/instructions/bundle.rs

```diff
@@ -188,5 +209,16 @@ pub fn execute_relayer_refund_leaf<'info>(
         caller: ctx.accounts.signer.key(),
     });
 
+    // Emit the DeferredRelayerRefunds event if any claim accounts were passed instead of token accounts. We cannot emit
+    // individual accounts and amounts as that would run out of memory for larger leaves.
+    if deferred_refunds {
+        emit_cpi!(DeferredRelayerRefunds {
```

Comment:
I think it's ok to emit all this but it's not really needed, right? we really just want to know that it happened, but care less about all the other props give we have `ExecutedRelayerRefundRoot`.

an alternative structure could simply be to add a bool to `ExecutedRelayerRefundRoot` that indicates if `deferred_refunds` is set so we don't need a. new event?

Reply:
yes, if we can change interface of `ExecutedRelayerRefundRoot` this would be much easier to avoid separate `emit_cpi`

Reply:
pushed the commit to include the bool in `ExecutedRelayerRefundRoot`